### DISCUSSION TITLE
Parameters for transcode, sidecar, and cash

### DISCUSSION THREAD

**Discussion by alikej** [photoprism/photoprism#1156] [Q&A] [answered]
Hi, I try to set up my new installation. My goal is to replace iCloud photos with cca 700GB of photos nad videos with this excellent app. I have some questions because I didn't find answers in docs.

1. The big challenge is HEVC 4K videos. When I imported a original file, the app transcoded it to .AVC file and stored it in sidecar. How can I set parameters for this transcoding because of storage (2x more space than original files?) and bandwidth? I intend to transcode all videos to max. 480p resolution for web streaming and original resolution use for download only.
2. How to set docker HW acceleration VAAPI on linux server (Unraid) for /dev/dri/renderD128 (intel xeon)? (this device i have mapped in docker composer)

My questions point to enviroments:  PHOTOPRISM_FFMPEG_ENCODER and PHOTOPRISM_FFMPEG_BIN and others

Thx for help.

**Comment by lastzero** [comment, marked as answer]
- Configuring the transcoding quality in such detail is not possible yet, see #1070
- Hardware acceleration is an advanced topic, which very much depends on your specific hardware, see #703 

You don't need to change `PHOTOPRISM_FFMPEG_BIN` if you're using our Docker image. `PHOTOPRISM_FFMPEG_ENCODER` may be changed to enable hardware transcoding. For more information on hardware encoders, the ffmpeg docs: https://trac.ffmpeg.org/wiki/HWAccelIntro
